### PR TITLE
KAB-27 Allow scoped token to create storage tokens

### PR DESCRIPTION
**Pull request by tomasfejfar**
Jira: KAB-27
Connection PR: https://github.com/keboola/connection/pull/4961

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

## Release

- [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

---

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ services:
       - KBC_MANAGE_API_SUPER_TOKEN_WITH_PROJECTS_READ_SCOPE
       - KBC_MANAGE_API_SUPER_TOKEN_WITH_DELETED_PROJECTS_READ_SCOPE
       - KBC_MANAGE_API_SUPER_TOKEN_WITHOUT_SCOPES
+      - KBC_MANAGE_API_SUPER_TOKEN_WITH_STORAGE_TOKEN_SCOPE
```
Comment (jirkasemmler) [inline docker-compose.yml:20]: prosim pridej a popis ho do README.md

**Comment by jirkasemmler** [comment]
jo a prosim chtelo by to pridat jeste sem https://keboolamanagementapi.docs.apiary.io/#reference/projects/create-storage-token-in-project/create-storage-token

File: docker-compose.yml (hunk 1 above)
Comment (tomasfejfar) [reply docker-compose.yml]: Done

Comment (tomasfejfar) [reply docker-compose.yml]: Apiary jsem měl upravené, ale zapomněl jsem pushnout 🤦

**Review by jirkasemmler — APPROVED** [review]

**Pull request merged**